packages/cli/src/oclif/commands/env/set.js

```diff
@@ -0,0 +1,86 @@
+const { cyan } = require('colors/safe');
+const { omit } = require('lodash');
+
+const BaseCommand = require('../../ZapierBaseCommand');
+const { buildFlags } = require('../../buildFlags');
+const { callAPI, getLinkedApp } = require('../../../utils/api');
+
+const successMessage = version =>
+  `Successfully wrote the following to the environment of version ${cyan(
+    version
+  )}:`;
+
+class SetEnvCommand extends BaseCommand {
+  async perform() {
+    const { version } = this.args;
+    this.throwForInvalidVersion(version);
+    // args should be [ '1.0.0', 'qer=123', 'qwer=123' ]
+    const valuesToSet = this.argv
+      .slice(1)
+      .filter(k => !k.startsWith('-'))
+      .map(k => k.toUpperCase());
```

Comment:
This also transforms the value into upper cases:

```
$ zapier env:set 1.0.0 hello=world
Successfully wrote the following to the environment of version 1.0.0:
{
  "HELLO": "WORLD"
}
```

We should probably delete the `toUpperCase` line here because it looks like you uppercase the environment key in the later code at line 37.